packages/plugin-e2e/src/types.ts

```diff
@@ -80,53 +80,6 @@ export type PluginFixture = {
    */
   selectors: E2ESelectors;
 
-  /**
```

Comment:
changing the order of the fixtures in the public api so that all pages are next to each other.